### PR TITLE
Update theme layouts and styles

### PR DESCRIPTION
**Pull request by adamwoodnz**
See #2 

This PR:

1. Updates content and images
2. Updates typography, colors and spacing
3. Removes unnecessary styles (IE8, supports, etc)

NOTE: content for What's On and Live is not functional yet and requires further work in #4 and #3, main nav and locale switcher requires #1, and Swag links are empty, pending #5. There are more refinements to be made to pagination and mobile views.

### Desktop

| What's on | News | Single | Live | Swag |
|-|-|-|-|-|
| localhost_8888_(Desktop) | localhost_8888_news_(Desktop) (1) | localhost_8888_wp20-commemorative-jones-soda-bottles-design-yours_(Desktop) (1) | localhost_8888_live_(Desktop) (1) | localhost_8888_swag_(Desktop) (1) |

### Mobile

| What's on | News | Single | Live | Swag |
|-|-|-|-|-|
| localhost_8888_(Samsung Galaxy S20 Ultra) (2) | localhost_8888_news_(Samsung Galaxy S20 Ultra) | localhost_8888_wp20-commemorative-jones-soda-bottles-design-yours_(Samsung Galaxy S20 Ultra) | localhost_8888_live_(Samsung Galaxy S20 Ultra) | localhost_8888_swag_(Samsung Galaxy S20 Ultra) |

## How to test

1. Follow the [setup instructions](https://github.com/WordPress/wp20.wordpress.net/tree/trunk#readme)
2. Navigate the pages (including single post); check layout against designs above, responsive behaviour, a11y.

### CODE REVIEW

File: wp-content/themes/twentyseventeen-wp20/page-news.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			'orderby'        => 'date',
+			'order'          => 'DESC',
+		) );
+		
+		$posts_by_month = array();
+		foreach ( $posts as $post ) {
+			$month = date( 'F', strtotime( $post->post_date ) );
+			$month = substr( $month, 0, 3 );
+			$posts_by_month[ $month ][] = $post;
+		}
```
Comment (adamwoodnz) [inline wp-content/themes/twentyseventeen-wp20/page-news.php]: I've refined this, but the initial suggestion for how to query and group was suggested by ChatGPT.

Let me know if there's a better way.

**Comment by adamwoodnz** [comment]
Remaining work:

- Swag content is not finished and there might be more styling required
- There is a bit of a glitch when the navigation becomes sticky. The content jumps up behind the nav, so I think extra padding needs to be applied to `.site-content` when the nav is sticky. This is also the case with wp15 so maybe it's not a big deal. I have seen the page layout glitching repeatedly when the content is just the right height to be in between states which isn't nice.

**Comment by StevenDufresne** [comment]
I updated the swag page, the only thing missing is the asset links. Do we have those assets somewhere?

 It would be nice if someone audited the swag page to make sure it's right.

**Comment by StevenDufresne** [comment]
> There is a bit of a glitch when the navigation becomes sticky. The content jumps up behind the nav, so I think extra padding needs to be applied to .site-content when the nav is sticky. This is also the case with wp15 so maybe it's not a big deal. I have seen the page layout glitching repeatedly when the content is just the right height to be in between states which isn't nice.

I fixed this in [968a217](https://github.com/WordPress/wp20.wordpress.net/pull/6/commits/968a217ec750dbc89ed6a2f249108cdc66754fd2). It's done using JS, not the heartiest of solutions but should work for this theme. Let me know.

**Comment by iandunn** [comment]
> It would be nice if someone audited the swag page to make sure it's right.

One approach that's worked before is to post screenshots and ping the designer here to review them.

> not the heartiest of solutions but should work for this theme

I agree, this is just a one-off theme 👍🏻

File: wp-content/themes/twentyseventeen-wp20/style.css [hunk 2]
```diff
@@ -313,7 +313,7 @@ body.page:not(.twentyseventeen-front-page) .entry-title {
 
 .navigation-top .current-menu-item > a,
 .navigation-top .current_page_item > a {
-	color: var(--wp20--color--text-primary);
+	color: var(--wp20--color--text);
```
Comment (StevenDufresne) [inline wp-content/themes/twentyseventeen-wp20/style.css:393]: Don't worry too much about the navigation. It needs some rework for it to match the designs on mobile. I'll take care of it in #7.

File: wp-content/themes/twentyseventeen-wp20/page-about.php [hunk 3]
```diff
@@ -7,26 +7,23 @@
...
-
-					<img class="wp20-confetti-divider" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/confetti-divider.svg" alt="" />
-
-					<p class="wp10-nostalgia">
-						<?php printf(
-							wp_kses_data( __( 'Check out <a href="%s">this post about the WordPress 10th anniversary</a>.', 'wp20' ) ),
-							'https://wordpress.org/news/2013/05/ten-good-years/'
-						); ?>
+					<div class="entry-content-section">
+						<?php echo do_shortcode( '[wp15_meetup_events]' ); ?>
```
Comment (iandunn) [inline wp-content/themes/twentyseventeen-wp20/page-about.php]:
```suggestion
						<?php echo do_shortcode( '[wp20_meetup_events]' ); ?>
```

I renamed it in 8b5dbc0caa0d9a80e8c1af02223e4982f435a899

**Comment by StevenDufresne** [comment]
I agree with @iandunn, can we merge this in? [ref](https://github.com/WordPress/wp20.wordpress.net/pull/7#issuecomment-1468563481)

**Comment by adamwoodnz** [comment]
> I agree with @iandunn, can we merge this in? [ref](https://github.com/WordPress/wp20.wordpress.net/pull/7#issuecomment-1468563481)

Yep I think it's in a good place now

File: wp-content/themes/twentyseventeen-wp20/page-whats-on.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?php get_header(); ?>
+
+<div class="wrap">
+	<div id="primary" class="content-area">
+		<main id="main" class="site-main" role="main">
+			<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+				<header class="entry-header">
+					<h1 class="entry-title">
+						<?php esc_html_e( 'People all over the world are celebrating the WordPress 20th Anniversary on May 27, 2022. Join us!', 'wp20' ); ?>
```
Comment (vlad-timotei) [inline wp-content/themes/twentyseventeen-wp20/page-whats-on.php:10]: Hi! I think this should be May 27, 2023 instead of 2022? @adamwoodnz

Comment (iandunn) [reply wp-content/themes/twentyseventeen-wp20/page-whats-on.php:10]: Hey 👋🏻 You're right, thanks for pointing that out. @timse201 fixed that in #25

**Pull request merged**